src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx

```diff
@@ -69,30 +72,31 @@ const ValidatorFlow = () => {
   function inputHandler(e: any) {
     const value = e.target.value;
     if (value > clusterBalance) {
-      setInputValue(clusterBalance);
+      setWithdrawValue(clusterBalance);
     } else if (value < 0) {
-      setInputValue(0);
+      setWithdrawValue(0);
     } else if (value === '') {
-      setInputValue(value);
+      setWithdrawValue(value);
     } else {
-      setInputValue(Number(value));
+      setWithdrawValue(Number(value));
     }
   }
 
   function maxValue() {
     // @ts-ignore
-    setInputValue(Number(clusterBalance));
+    setWithdrawValue(Number(clusterBalance));
   }
   
-  const newBalance = (inputValue ? clusterBalance - Number(inputValue) : clusterBalance).toFixed(18);
+  const newBalance = (withdrawValue ? clusterBalance - Number(withdrawValue) : clusterBalance).toFixed(18);
   // @ts-ignore
   const errorButton = clusterStore.getClusterRunWay({ ...cluster, balance: walletStore.toWei(newBalance) }) <= 0;
   const showCheckBox = clusterStore.getClusterRunWay({ ...cluster, balance: walletStore.toWei(newBalance) }) <= 30;
+  const buttonDisableCondition = clusterStore.getClusterRunWay({ ...cluster, balance: walletStore.toWei(newBalance) }) <= 30 && !userAgree || Number(withdrawValue) === 0 || !checkedCondition;
```

Comment:
What is this 30 constant, can you move it to global constants in config.ts file if it's important to not lose